website/src/pages/chat/[id].tsx

```diff
@@ -47,9 +53,15 @@ export const getServerSideProps: GetServerSideProps<ChatProps, { id: string }> =
       notFound: true,
     };
   }
+
+  const { data: modelInfos } = await axios.get<ModelInfo[]>("/configs/models", {
+    baseURL: process.env.INFERENCE_SERVER_HOST,
+  });
+
```

Comment:
I wonder why do we do this server side?

Reply:
SSR will make the code much cleaner. Fetching data in the client will require calling `setValue` on success to update the default form value. Also, we don't have to check where it's loading or not.